src/db.h

```diff
@@ -141,6 +141,17 @@ enum data_kind {
 const char *
 db_data_kind_label(enum data_kind data_kind);
 
+
+/* Indicates user marked status on a track */
+enum flag_kind {
```

Comment:
As usual, naming is difficult, and I can see now that your "'review_flag" may have been better. "flag_kind" is inconsistent because it isn't called that in the db, "flag" is too generic for something that has effectively global scope, and "db_flag" is inconsistent with the other enums that don't have DB_ prefixed. Of course, "REVIEW_FLAG_REVIEW" is also a bit odd. So no straightforward alternative. Inspired by the commend you added above maybe "user_mark"? - how does that sound to you?